addons/l10n_ro/data/account_tax_report_data.xml

```diff
@@ -6,405 +6,984 @@
         <field name="country_id" ref="base.ro"/>
     </record>
 
-    <record id="account_tax_report_baza_tva_clt" model="account.tax.report.line">
-        <field name="name">BAZA TVA COLECTAT</field>
+    <record id="account_tax_report_ro_baza_intracom_eu" model="account.tax.report.line">
+        <field name="name">Baza COMERŢ INTRACOMUNITAR ŞI ÎN AFARA UE</field>
+        <field name="code">tax_ro_baza_intracom_eu</field>
```

Comment:
@jco-odoo I would say no, since the previous tax report was not compliant with all the regulations, that's why here we have the biggest changes, before it was basically a copy of the taxes, but the new one is based on the tax report that you will have to supply to the fiscal authorities.

Reply:
@feketemihai Cool, but we will need a mapping between the old tax report lines and the new for those who migrate.

Reply:
@jco-odoo Did you had this before, what's the best way:
 - rename the xml_ids for previous tax report lines (account.tax.report.line), and on update they will be overwritten with new config, but the previous repartition lines will remain
 or
- replace the tag_ids in the account.tax.repartition.line model from old one's to new one's

Reply:
None of those.  It has to happen in migration and for that I just need to know which old lines correspond to which new lines.

Reply:
I will check with @rgo-odoo

Reply:
@jco-odoo probably on monday-tuesday i can try to do a mapping, and let you know